packages/lo/lo.0.1.2/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-lo"
+build: [
+  ["./configure" "--prefix" prefix] {os != "macos"}
+  [
+    "./configure"
+    "CFLAGS=-I/usr/local/include"
+    "LDFLAGS=-L/usr/local/lib"
+    "OCAMLFLAGS=-ccopt -I/usr/local/include -cclib -L/usr/local/lib"
+    "--prefix"
+    prefix
+  ] {os = "macos"}
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "lo"]
+depends: ["ocaml" "ocamlfind"]
```

Comment:
```suggestion
depends: ["ocaml" "ocamlfind" {build}]
```

I did not check the code, do you need ocaml find at runtime?

Reply:
No need for ocamlfind at runtime, merging your patch.